test/e2e/commands/selectOutputFrame.js

```diff
@@ -0,0 +1,4 @@
+exports.command = function(number) {
```

Comment:
This command should take no arguments and should always select 2nd frame, otherwise it's no different from frame command.